api/turing/validation/validator_test.go

```diff
@@ -645,6 +645,122 @@ func TestValidateTrafficRules(t *testing.T) {
 				"'Fallback Route (DefaultRouteId): 'route-a' should be associated to all Traffic Rules' tag",
 			}, " "),
 		},
+		"failure | Overlapping Traffic Rules": {
```

Comment:
Can we also add a test case with multiple rules and different combinations of the conditions that result in a success?